packages/contracts/contracts/governance/UpgradeGovernor.sol

```diff
@@ -120,6 +120,41 @@ contract UpgradeGovernor is
         return super.proposalThreshold();
     }
 
+    /**
+     * @notice Function to queue a proposal to the timelock.
+     *         Added protocol for using custom time-lock zero delay for urgent situations.
+     */
+    function queue(
```

Comment:
Isn't it better to place this function above the comment `// The following functions are overridden cause required by Solidity.` in L67?